src/main.ts

```diff
@@ -19,6 +19,9 @@ const internalExternsFileName = 'tsickle_externs.js';
 interface Settings {
   /** If provided, path to save externs to. */
   externsPath?: string;
+
+  /** If provided, convert every type to the Closure {?} type */
+  isTyped: boolean;
```

Comment:
Nice catch. That's what I get for copy pasting. I'll change it to untyped to keep it the same as the tsickle library.